src/components/AuthSettings.tsx

```diff
@@ -48,19 +55,19 @@ export default function AuthSettings(): ReactElement {
                 credentialProfiles: undefined,
               })
             );
+            location.reload();
           }}
         >
           {(props) => (
             <div className="space-y-6 sm:px-6 lg:px-0 lg:col-span-9">
-              <form onSubmit={props.handleSubmit} noValidate>
+              <form onSubmit={props.handleSubmit} noValidate id="test">
```

Comment:
Not saying we don't need an id here, just wondering what it is used for. If we want to add it, make sure it represents the element. e.g. `settings-form`